src/database/schema/ChatSchema.ts

```diff
@@ -158,13 +164,20 @@ class ChatSchema implements BaseDbSchema {
 
   static fromDatabaseObject(dbObject: any): ChatSchema {
     let rawJson = null;
+    let attachmentJson = null;
 
     try {
       rawJson = JSON.parse(dbObject?.raw_json || '{}');
     } catch (e) {
       console.log('error parse');
       console.log(e);
     }
+    try {
+      attachmentJson = JSON.parse(dbObject?.attachment_json || '[]');
+    } catch (e) {
+      console.log('error parse');
+      console.log(e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You are parsing `raw_json` and `attachment_json` from the database object and logging errors if they fail. However, these logs do not provide enough context about what went wrong. Consider adding more information to the error logs, such as the id of the chat message that caused the error.



<!-- This is an auto-generated comment by OSS CodeRabbit -->